### ISSUE TITLE
Movable Card mouseover image

### ISSUE THREAD

**Issue by SEforbinproject** [MetaMorphic-Digital/torgeternity#131]
A useful feature would be the ability to set or move where the mouse over image for cards actually appears. Right now it appears right over the Drama card and it would be an improvement if it could be set to a less obtrusive place by the GM and by each player.

**Comment by rwanoux** [comment]
The cards can be hovered from different Ui elements. Could you precise from which window or is it from the chat that you would like to move the zoomed card.

**Comment by SEforbinproject** [comment]
From the pool section for each storm knight in the combat tracker; the mouse over window tends to obscure the drama card

**Comment by Algor0113** [comment]
Doesn't it vanish when you quit the mouseover? @SEforbinproject

**Comment by pemerson71** [comment]
@SEforbinproject , if the card is not disappearing for you when you remove your mouse from that area, so you can clearly see the Drama card in the CT, again, please let us know, as that would be a potential bug?

**Comment by SEforbinproject** [comment]
So here's a screenshot of how things appear in the combat tracker: 
![image](https://user-images.githubusercontent.com/69514528/174441792-d59132e9-5269-40cc-835f-f549cd5af945.png)

**Comment by SEforbinproject** [comment]
So it does go away when I move off the card name, but that's a weird place for the card to appear.

**Comment by pemerson71** [comment]
Okay, the cards in my CT always show to the upper-right of the onHover link and, thus, as I'm looking through pool cards, the card representations move, as well, when I move to the next card.  If yours ONLY pops up over the top of the Drama Card, that could be a problem.  Otherwise, this is working as advertised.

If you want the card to move with your cursor, as though you're dragging it around, I'm not sure that's possible.

If you want to be able to see the Drama card and the Pool card popup, at the same time... why?

Forgive me if it sounds as though I'm being obstinate, as I am not; however, before the TGS team attempts to go through a whole bunch of work that is, potentially, unnecessary, to solve what most will see as a minor problem, is the reason I'm asking the questions?

Would you prefer the card showed at the upper-left or lower-left of the onHover link for the Pool cards?

**Comment by pemerson71** [comment]
Looking at the screenie you dropped earlier in this thread, it appears as though the Pool card is popping up over top of the Drama card and is very far away from the onHover link for the card.  It doesn't do that in my system, so I'm thinking you have a module installed, somewhere, or your video adapter device is set to a small resolution, that is making the card show up that far away.  Are you running at least at the Foundry minimum resolution?

**Comment by SEforbinproject** [comment]
So here's a completely vanilla system, no modules at all; 
![image](https://user-images.githubusercontent.com/69514528/174444614-dc7659f2-be3d-4b89-9664-26b8396ebdc4.png)

**Comment by SEforbinproject** [comment]
Ok, so why I raised this is kind of a UX/gm'ing thing. I think to consider how some cards might interact with the current Drama Card in interesting ways, or how I might describe things in the context of the current Drama card. So I'm cognitively wanting to compare two things, which is a pain when one overlaps the other. I realize this is probably super-niche, and I have no expectation is should necessarily get on the list given how many other things you've got to look at. For me, I'm always looking at ways in which I can tweak information access for both me and the players, hence my interest in this issue.

**Comment by gmmatt** [comment]
Just a quick comment on why this is what it is. The problem that we have run into is that the CSS technique that we use to display the tooltip image will not work in a pop out unless the tooltip remains inside the combat tracker. I actually experimented quite a lot with different locations within the combat tracker for the tool tip. I ended up putting it near the top because I wanted to minimize the risk that the tooltip would cover up the location where the mouse was hovering.

This is not ideal. I would love to have a tooltip that appears beside the combat tracker instead of inside the combat tracker, even when popped out. If there’s anybody out there  who has some experience building more sophisticated tooltips, I would love to hear from you.
